crates/turbopack-browser/src/chunking_context.rs

```diff
@@ -133,6 +136,7 @@ impl BrowserChunkingContext {
         asset_root_path: Vc<FileSystemPath>,
         environment: Vc<Environment>,
         runtime_type: RuntimeType,
+        global_information: Vc<OptionGlobalInformation>,
```

Comment:
That shouldn't be passed here (we don't want to force to always pass that).

Instead add a `global_information()` method to pass that, similar to the other optional fields